paasta_tools/cli/cmds/local_run.py

```diff
@@ -577,10 +583,6 @@ def run_docker_container(
             for line in docker_client.attach(container_id, stderr=True, stream=True, logs=True):
                 sys.stdout.write(PaastaColors.grey(line))
         else:
```

Comment:
You still need to print the output if there is no healthcheck_mode and the container isn't running.